TestLogger (persist module) unit test times out

Master build status is currently broken, due to a timed out unit test.

From https://travis-ci.org/rivine/rivine/jobs/347796187:

```
panic: test timed out after 5s
goroutine 15 [running]:
testing.startAlarm.func1()
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/testing/testing.go:1023 +0xf9
created by time.goFunc
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/time/sleep.go:170 +0x44
goroutine 1 [chan receive]:
testing.(*T).Run(0xc4200628f0, 0x5bf813, 0xa, 0x5ca058, 0xc4200afd01)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/testing/testing.go:698 +0x2f4
testing.runTests.func1(0xc4200628f0)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/testing/testing.go:882 +0x67
testing.tRunner(0xc4200628f0, 0xc4200afde0)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/testing/testing.go:657 +0x96
testing.runTests(0xc42000b120, 0x696aa0, 0xd, 0xd, 0x5ade80)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/testing/testing.go:888 +0x2c1
testing.(*M).Run(0xc4200aff20, 0xc4200aff20)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/testing/testing.go:822 +0xfc
main.main()
	github.com/rivine/rivine/persist/_test/_testmain.go:78 +0xf7
goroutine 14 [syscall]:
syscall.Syscall(0x4a, 0x1c, 0x0, 0x0, 0x6b4758, 0x101, 0x6b6d80)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.Fsync(0x1c, 0x4d, 0x554c35)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/syscall/zsyscall_linux_amd64.go:492 +0x4a
os.(*File).Sync(0xc42000c180, 0x5ca6c8, 0xc4200174ac)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/os/file_posix.go:121 +0x5b
github.com/rivine/rivine/persist.(*closeableFile).Close(0xc4200174a0, 0x0, 0x0)
	/home/travis/gopath/src/github.com/rivine/rivine/persist/log.go:101 +0x96
github.com/rivine/rivine/persist.(*Logger).Close(0xc42000b540, 0xc42004eec8, 0x1)
	/home/travis/gopath/src/github.com/rivine/rivine/persist/log.go:26 +0x94
github.com/rivine/rivine/persist.TestLogger(0xc4200632b0)
	/home/travis/gopath/src/github.com/rivine/rivine/persist/log_test.go:32 +0x2d7
testing.tRunner(0xc4200632b0, 0x5ca058)
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/testing/testing.go:657 +0x96
created by testing.(*T).Run
	/home/travis/.gimme/versions/go1.8.7.linux.amd64/src/testing/testing.go:697 +0x2ca
FAIL	github.com/rivine/rivine/persist	18.723s
```

--- a/persist/log_test.go
+++ b/persist/log_test.go
@@ -13,6 +13,10 @@ import (
 // TestLogger checks that the basic functions of the file logger work as
 // designed.
 func TestLogger(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+
 	// Create a folder for the log file.
 	testdir := build.TempDir(persistDir, t.Name())
 	err := os.MkdirAll(testdir, 0700)

